__tests__/src/index.spec.js

```diff
@@ -135,20 +119,21 @@ describe('toMatchImageSnapshot', () => {
         added: true,
       },
     };
-    const mockDiffResult = { updated: true, code: 7 };
+    const mockDiffResult = { updated: false, pass: true };
 
     setupMock(mockDiffResult);
     const { toMatchImageSnapshot } = require('../../src/index');
     const matcherAtTest = toMatchImageSnapshot.bind(mockTestContext);
 
-    matcherAtTest('pretendthisisanimagebuffer');
+    const result = matcherAtTest('pretendthisisanimagebuffer');
     const { diffImageToSnapshot } = require('../../src/diff-snapshot');
 
     const dataArg = diffImageToSnapshot.mock.calls[0][0];
     // This is to make the test work on windows
     dataArg.snapshotsDir = dataArg.snapshotsDir.replace(/\\/g, '/');
 
     expect(dataArg).toMatchSnapshot();
+    expect(result.message()).toMatchSnapshot();
```

Comment:
whats the point of this assertion here?

Reply:
It always felt dirty to me that previously a test pass would still return the failure message so I modified it so that a pass will return a function returning an empty string, the fail message is only return on a fail. 

However the result is that I need to perform an assertion on that function or it fails on the coverage.